packages/operators/src/common/switch.js

```diff
@@ -0,0 +1,42 @@
+/*
+  Copyright 2020-2021 Lowdefy, Inc
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import { type } from '@lowdefy/helpers';
+
+function _switch({ location, params }) {
+  if (!type.isArray(params.branches)) {
+    throw new Error(
+      `Operator Error: switch takes an array type as input for the branches. Received: ${JSON.stringify(
+        params
+      )} at ${location}.`
+    );
+  }
+  for (let i = 0; i < params.branches.length; i++) {
```

Comment:
Please change to [`for.. ...of`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) syntax.

```
for (const branch of  params.branches) {
```